clang/lib/CIR/CodeGen/CIRGenBuiltinAArch64.cpp

```diff
@@ -2368,10 +2369,20 @@ mlir::Value CIRGenFunction::buildCommonNeonBuiltinExpr(
                        : "llvm.aarch64.neon.srhadd";
     break;
   }
+  case NEON::BI__builtin_neon_vqmovun_v: {
+    intrincsName = "llvm.aarch64.neon.sqxtun";
+    argTypes.push_back(builder.getExtendedOrTruncatedElementVectorType(
+        vTy, true /* extended */, true /* signed */));
+    break;
+  }
+  }
+
+  if (!intrincsName.empty()) {
+    if (argTypes.empty())
```

Comment:
Please add a short comment explaining why `{vTy, vTy}` when empty. Alternatively, perhaps buildCommonNeonCallPattern0 should create `{vTy, vTy}` when `argTypes` is empty? Feels like this logic should be inside that function instead (but I don't have the full context).

Reply:
I will let buildCommonNeonCallPattern0 create {vTy, vTy} when argTypes is empty to avoid confusion on the call site.
{vTy, vTy} used as default cuz its most common (so far as I can see) set of arg types.